src/components/UserInfo.js

```diff
@@ -1,21 +1,24 @@
 export default class UserInfo {
-  constructor(nameSelector, jobSelector) {
+  constructor(nameSelector, jobSelector, avatorselector) {
     this._name = document.querySelector(nameSelector);
     this._job = document.querySelector(jobSelector);
+    this._avatar = document.querySelector(avatorselector);
   }
 
   //   returns an object with information about the user to display the user data in the open form.
   getUserInfo() {
     this._userInfo = {
       name: this._name.textContent,
-      description: this._job.textContent,
+      about: this._job.textContent,
+      avatar: this._avatar.src,
     };
     return this._userInfo;
   }
 
   //   takes new user data and adds it on the page.
-  setUserInfo(nameInput, jobInput) {
-    this._name.textContent = nameInput;
-    this._job.textContent = jobInput;
+  setUserInfo(obj) {
+    this._name.textContent = obj.name;
+    this._job.textContent = obj.about;
+    this._avatar.src = obj.avatar;
```

Comment:
Needs correction: It's needed to create a separate setter for avatar `setAvatar` and use it when needed to update the avatar, because user info and avatar are controlled by different forms.

When initializing user info you can use bit methods:

```js
userInfo.setUserInfo({ title, description })
userInfo.setAvatar(avatar)
```